### PR TITLE
[PPP-4751] updated active mq to 5.16.7

### PR DESCRIPTION
**Pull request by MarcDiamantHitachi**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -1290,6 +1290,11 @@
         <artifactId>snappy-java</artifactId>
         <version>${snappy-java.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.activemq</groupId>
+        <artifactId>activemq-all</artifactId>
+        <version>activemq.version</version>
```
Comment (lucboudreau) [inline pom.xml:1296]: You're missing the ${..} brackets here

**Review by lucboudreau — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 15s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by smmribeiro — APPROVED** [review]
Looks good!

**Pull request merged**